woocommerce/woocommerce-gutenberg-products-block
# Move `RadioControl`and `RadioControlAccordion` components to components package

## What
This PR is based on https://github.com/woocommerce/woocommerce-blocks/pull/11214 (update/checkbox-component)
Move `RadioControl` and `RadioControlAccordion` components to components package and update the places it is imported:

- ShippingRates
- PaymentMethods
- PickupOptions
- LocalPickup

## Why




## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Install [Multiple Packages for WooCommerce](https://wordpress.org/plugins/multiple-packages-for-woocommerce/) and go to WooCommerce > Settings > Multiple Packages. Set the "Group by" option to Product (individual).
2. Create at least 2 pickup locations (WooCommerce > Settings > Shipping > Local Pickup)
3. Add multiple products to your cart. Ensure you add one product more than once (so you have 2+ of a single item in the cart)
4. Visit the checkout
5. In the shipping options section, switch between different options and make sure the right option is selected and they are greyed out while the http request is in progress.
6. In the shipping method section, switch to Local Pickup
7. Select one of the other pickup locations and make sure the right option is selected.
8. Scroll down to the Payment Options section.
9. Change the payment method and make sure the right option is selected.
10. Fill in all the other details, place the order and make sure it goes through ok.


## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Make the `RadioControl` and `RadioControlAccordion` component available in the @woocommerce/blocks-components package.
